build.gradle

```diff
@@ -166,13 +166,18 @@ subprojects { proj ->
     }
 
     if (proj.documentationProject.toBoolean()) {
-        apply plugin: 'org.asciidoctor.convert'
+        apply plugin: "org.asciidoctor.jvm.convert"
+
+        repositories {
```

Comment:
repositories are already defined in global part. Is a 2nd definition really needed?